source/libtomo/accel/gpu/mlem.cu

```diff
@@ -156,7 +157,7 @@ mlem_gpu_compute_projection(data_array_t& gpu_data, int p, int dy, int dt, int d
         CUDA_CHECK_LAST_STREAM_ERROR(stream);
 
         // synchronize the stream (do this frequently to avoid backlog)
-        stream_sync(stream);
+        // stream_sync(stream);
```

Comment:
Is `stream_sync` supposed to be disabled here?

Reply:
Yes. Based on that comment and my recollection, that stream sync was a by-product of a bottleneck of too many kernel launches getting queued up in the stream. No longer the case since the introduction of processes reduced the contention for submitting kernel launches.